src/fiskaltrust.Middleware.Interface.Client.Http/ESSSCD/HttpESSSCDClientOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace fiskaltrust.Middleware.Interface.Client.Http
+{
+    /// <summary>
+    /// Used to pass client options to the underlying HTTP client
+    /// </summary>
+    public class HttpESSSCDClientOptions : ClientOptions
+    {
+        public bool? DisableSslValidation { get; set; }
```

Comment:
```suggestion
        public bool? DisableSslValidation { get; set; }
        public Guid? CashboxId { get; set; }
        public string? AccessToken { get; set; }
```

If they are set we'll need to add those also as `x-` headers on the requests we send so that the signing-es can authenticate the request.